src/main.rs

```diff
@@ -268,5 +255,6 @@ fn main() {
         ("led", Some(m)) => DiskShelf::jbod_led_switch(m),
         ("prometheus", Some(m)) => fork_prometheus(m),
         _ => Ok(help()),
-    };
+    }
+    .expect("jbod encountered an internal issue:")
```

Comment:
Would also print the `Err`.

Reply:
Something like: 
[playground link](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&code=use%20std%3A%3Aio%3A%3AWrite%3B%0Afn%20main()%20%7B%0A%20%20%20%20%22toto%22.parse%3A%3A%3Ci32%3E().expect(%22%3CInternal%20Error%3E%22)%3B%0A%7D)
```rust
fn main() {
    "toto".parse::<i32>().expect("<Internal Error>");
}
```
```
thread 'main' panicked at '<Internal Error>: ParseIntError { kind: InvalidDigit }', src/main.rs:3:27
note: [run with `RUST_BACKTRACE=1` environment variable to display a backtrace](https://play.rust-lang.org/#)
```